src/abstractdataframe/io.jl

```diff
@@ -165,6 +165,59 @@ end
 
 ##############################################################################
 #
+# LaTeX output
+#
+##############################################################################
+
+function latex_char_escape(char::SubString)
```

Comment:
It's my understanding that `SubString` will eventually be merged into `String`. Also, why not use `Char`?